src/statement.d

```diff
@@ -3787,7 +3787,7 @@ public:
             }
             sw.cases.push(this);
             // Resolve any goto case's with no exp to this case statement
-            for (size_t i = 0; i < sw.gotoCases.dim;)
+            foreach (i; 0 .. sw.gotoCases.dim)
```

Comment:
I think `foreach (ref i; 0 .. sw.gotoCases.dim)` will do what you want, but I'm not sure that's in 2.067.